src/Authentication/Traits/HasHmacTokens.php

```diff
@@ -156,4 +160,67 @@ public function setHmacToken(?AccessToken $accessToken): self
 
         return $this;
     }
+
+    /**
+     * Checks if the provided Access Token is expired.
+     *
+     * @return bool Returns true if Access Token is expired, false if not
```

Comment:
This comment is unnecessary as it can be inferred from the method description what it does.
```suggestion
```